README.md

```diff
@@ -1,9 +1,14 @@
 Authors Haven - A Social platform for the creative at heart.
 =======
+# Develop Branch badges
 [![Build Status](https://travis-ci.org/andela/ah-backend-spaces.svg?branch=develop)](https://travis-ci.org/andela/ah-backend-spaces)
 [![Coverage Status](https://coveralls.io/repos/github/andela/ah-backend-spaces/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-backend-spaces?branch=develop)
 [![Maintainability](https://api.codeclimate.com/v1/badges/96588e5cfb3ba7da70bb/maintainability)](https://codeclimate.com/github/andela/ah-backend-spaces/maintainability)
 
+# ch-travisci-badge-159082370 branch badges
```

Comment:
Why do we have the badges here too

I believe it should be 'develop branch's badges only

Reply:
Just for testing